plugin.js

```diff
@@ -45,6 +47,162 @@ const globals = new Set([
   '_scrollTo',
 ]);
 
+const blacklistedFunctions = new Set([
+  'stopCapturing',
```

Comment:
maybe we should leave a note about this

Reply:
What would be a proper place for that?

Reply:
line 49 could be ok, like explaining the constant